book/core_programming.md

```diff
@@ -235,12 +235,13 @@ To summarise:
 
 ### Scripts
 
-Scripts are simply files containing code that you would like to execute. In Python you commonly have a `main.py` script that orchestrates part of your codebase to achieve an outcome. In machine-learning projects, you sometimes have `train.py` and `test.py` which are scripts that train the model and produce performance metrics.
+Scripts are simply files containing code that you would like to execute. In Python you commonly have a `main.py` script that orchestrates part of your codebase to achieve an outcome. In machine-learning projects, you sometimes have `train.py` and `test.py`, which are scripts that train the model and produce performance metrics.
 
 Scripts, if written well, are reproducible. In languages like R and Python, when executed from the command line using commands like `python main.py` they are read top to bottom and executed line by line. This is in contrast to running code in an interactive interpreter or notebook, where the human has control of the order of execution. Manually executing individual lines of code allows for a slew of errors when things are run in the wrong order.
 
 Ultimately for code pipelines you will need to have some way of running your code - the humble script is the primary way of orchestrating your functions and classes in a pipeline fashion.
 
+<!-- The first sentence of the note below doesn't make sense to me. -->
 ```{note}
 Including something that is not reproducible in a script will not make it more reproducible. The script is simply a tool to run code in the same fashion across multiple runs.
```

Comment:
Yes, I like that change. Too often programmers think their code is reproducible (myself included) without fully considering the steps that would still be necessary for someone to understand and replicate our work **but** using a programming script rather than manual steps is a massive step in the right direction!